src/ast.js

```diff
@@ -219,6 +219,46 @@ export class FunctionDefinition extends ASTNode {
   }
 }
 
+export class CondClause extends ASTNode {
+  constructor(from, to, testExpr, thenExprs, options={}) {
+    super(from, to, 'condClause', options);
+    this.testExpr = testExpr;
+    this.thenExprs = thenExprs;
+  }
+
+  *[Symbol.iterator]() {
+    yield this;
+    yield this.testExpr;
+    yield* this.thenExprs;
+  }
+
+  toString() {
+    return `[${this.testExpr} ${this.thenExprs.join(' ')}]`;
+  }
+}
+
+export class CondExpression extends ASTNode {
+  constructor(from, to, clauses, options={}) {
+    super(from, to, 'condExpression', options);
+    this.clauses = clauses;
+  }
```

Comment:
Are there tests somewhere that make sure all ast nodes have navigation pointers?